tests/monitoring/monitoring.go

```diff
@@ -21,6 +21,7 @@ package monitoring
 
```

Comment:
```
tests/monitoring/monitoring: Replace update with Patch
Signed-off-by: avlitman <alitman@redhat.com>
```
In your commit message, please explain *why* this commit is important. What is wrong with repeatedly Updating?

Reply:
Done